### PR TITLE
add distrax

### LINKED ISSUE #23157: Package request: distrax
### Package name

distrax

### Package version

Newest

### Package website

https://github.com/deepmind/distrax

### Package availability

https://pypi.org/project/distrax/

### Additional comments

This package is a dependency for https://github.com/deepmind/kfac-jax

### Package is not available

- [X] The package is not available on conda-forge.

### No previous issues or open PRs

- [X] No previous issue exists and no PR has been opened.

### PR DESCRIPTION
**Pull request by ngam**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Fix #23157

### CODE REVIEW

File: recipes/distrax/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - jax >=0.4
+    - jaxlib >=0.4
+    - numpy >=1.23.0
+    - tensorflow-probability >=0.15.0
+
+test:
+  imports:
+    - distrax
+    - examples
+    - requirements
```
Comment (awvwgk) [inline recipes/distrax/meta.yaml:34]: The directories examples and requirements should not be installed here.

File: recipes/kfac-jax/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - distrax >=0.1.3
+    - jax >=0.4.7
+    - jaxlib >=0.4.7
+    - dm-tree >=0.1.7
+    - optax >=0.1.4
+    - typing-extensions >=4.2.0
+
+test:
+  imports:
+    - docs
```
Comment (awvwgk) [inline recipes/kfac-jax/meta.yaml:34]: There is an additional docs directory installed here which should not be included

**Review by awvwgk — COMMENTED** [review]
Thanks for creating the recipes. I noticed some additional directories which get installed and shouldn't be included in the final package.

**Comment by fl-ferr** [comment]
Hi @ngam, thanks a lot for working on this recipe! I am also interested in bringing `distrax` in conda-forge as well, are you still interested in working on this PR? If not, I have a file ready to open a new PR to add `distrax`. Thanks a lot in advance!

**Pull request closed without merging**